app/components/bulk_actions_form_component.html.erb

```diff
@@ -168,11 +168,10 @@
     </div>
 
     <div data-bulk-actions-target="commonFields" >
-      <% if @last_search %>
+     <% unless search_params.blank? %>
         <button class='btn btn-primary' data-action="click->bulk-actions#populateDruids">
```

Comment:
hide this button if we don't have any incoming search parameters in the query string